docs/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright (c) 2018 Philip Diffenderfer
```

Comment:
@mraza007 Can you update this line to `Copyright (c) 2018 Email Dashboard`. Just updated in master to this as well.

Reply:
Sure I will do that